ospfd/ospf_zebra.c

```diff
@@ -1033,7 +1033,8 @@ static int ospf_zebra_read_route(ZAPI_CALLBACK_ARGS)
 		char buf_prefix[PREFIX_STRLEN];
 		prefix2str(&api.prefix, buf_prefix, sizeof(buf_prefix));
 
-		zlog_debug("%s: from client %s: vrf_id %d, p %s", __func__,
+		zlog_debug("%s: cmd %s from client %s: vrf_id %d, p %s",
```

Comment:
Should we open up a PR with just this commit as that it's completely non-controversial and should be in the code sooner rather than later?

Reply:
Yes, that's fine - I've removed this debug change and opened a separate PR for it.